staging/src/k8s.io/sample-apiserver/artifacts/example/deployment.yaml

```diff
@@ -20,8 +20,8 @@ spec:
       - name: wardle-server
         # build from staging/src/k8s.io/sample-apiserver/artifacts/simple-image/Dockerfile
         # or
-        # docker pull gcr.io/kubernetes-e2e-test-images/sample-apiserver:1.17
-        # docker tag gcr.io/kubernetes-e2e-test-images/sample-apiserver:1.17 kube-sample-apiserver:latest
+        # docker pull k8s.gcr.io/e2e-test-images/sample-apiserver:1.17.4
+        # docker tag k8s.gcr.io/e2e-test-images/sample-apiserver:1.17.4 kube-sample-apiserver:latest
```

Comment:
/cc @liggitt @lavalamp @deads2k 
we need an /approve from someone in sample-apiserver/OWNERS for this comment change